kubernetes/client/clientset/clientset.go

```diff
@@ -30,8 +43,8 @@ func (c *Clientset) ComposeV1beta1() composev1beta1.ComposeV1beta1Interface {
 	return c.ComposeV1beta1Client
 }
 
-// Deprecated: Compose retrieves the default version of ComposeClient.
-// Please explicitly pick a version.
+// Compose retrieves the default version of ComposeClient.
+// deprecated: please explicitly pick a version.
 func (c *Clientset) Compose() composev1beta1.ComposeV1beta1Interface {
```

Comment:
Hum even if it's deprecated, should we return `ComposeV1beta2Client` here instead :angel:

Reply:
Right

Reply:
Or I may remove it...